src/Microsoft.Identity.Web.Certificate/DefaultCertificateLoader.cs

```diff
@@ -74,6 +75,29 @@ public static string? UserAssignedManagedIdentityClientId
 
             return certDescription?.Certificate;
         }
+        
+        /// <summary>
+        /// Load the first certificate from the certificate description list.
+        /// </summary>
+        /// <param name="certificateDescriptions">Description of the certificates.</param>
+        /// <returns>First certificate in the certificate description list.</returns>
+        public static async Task<X509Certificate2?> LoadFirstCertificateAsync(IEnumerable<CertificateDescription> certificateDescriptions)
+        {
+            DefaultCertificateLoader defaultCertificateLoader = new(null);
+            CertificateDescription? certDescription = new();
```

Comment:
not sure why you don't initialize it to null?

Reply:
good call